generator-prisma-client/esbuild-cjs/README.md

```diff
@@ -0,0 +1,134 @@
+# Prisma Postgres Example: ESBuild + Node.js + CJS
+
+This project showcases how to use the Prisma ORM with Prisma Postgres in an TypeScript application transpiled to CommonJS (CJS) using ESBuild.
+
+## Prerequisites
+
+To successfully run the project, you will need the following:
+
+- Two **Prisma Postgres** connection strings:
+  - Your **Prisma Postgres + Accelerate connection string** (containing your **Prisma API key**) which you can get by enabling Postgres in a project in your [Prisma Data Platform](https://pris.ly/pdp) account. You will use this connection string to run Prisma migrations.
```

Comment:
I think the **Prisma Postgres + Accelerate connection string** is used with Prisma Client and not to run migrations. So the description should be swapped.

Reply:
This string is set to `DATABASE_URL` value so this will be used to interact with Prisma Client.